src/applications/mhv-landing-page/sass/mhv-landing-page.scss

```diff
@@ -14,6 +14,17 @@ $anchor-font-size: 1.25rem; // == 20px
   }
 }
 
+$mhv-desktop-screen: 1008px;
```

Comment:
Why the 1008? I wonder if we can use the already defined breakpoints in the VADS

Reply:
Ok. I'll played with it a little bit and I did not see much change from using 1024. I don't know the answer, but I am wondering if we should just use 1024. Maybe @dcloud or @radavis can chime it.

Reply:
Actually, `small-desktop-screen` is being used in Nav Card, so I wonder if using 1008 will be a mismatch

Reply:
That's partly the reason for using custom classname with 1008px breakpoint - to stay consistent with the [existing formation](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/4e95d2ad8bf8e9af532e6101b38a82d89f000c53/packages/formation/sass/base/_b-breakpoints.scss#L51) `small-desktop-screen` variable of 1008 since we used this classname in Nav Card as well.

Reply:
I see that the `small-desktop-screen` value of 1008px is still there in spite of the breakpoint changes in VADS. 1008px doesn't correspond to any desktop size or fraction (e.g., someone's browser window is 50% of the width of their screen). 

I guess I'm not sure if this is necessary or not, given the tag will have to wrap at some point, and the ticket only says "desktop layouts", leaving this open to broad interpretation.